src/components/main/AppCard.tsx

```diff
@@ -39,77 +41,58 @@ const AppCard = ({
   onClick,
   selected = false,
 }: Props): JSX.Element => (
-  <Card
-    sx={{
-      width: '100%',
-      outline: selected ? '2px solid #5050d2' : '',
-    }}
-    onClick={onClick}
-    id={buildItemFormAppOptionId(id ?? name)}
-  >
-    <StyledCardActionArea>
-      <Grid2
-        container
-        direction="row"
-        alignItems="center"
-        width="100%"
-        spacing={[0, 1]}
-      >
-        <Grid2 xs={2} p={[0, 1.5]}>
-          <Box
-            component="img"
-            src={image ?? defaultImage}
-            alt={name}
-            width="100%"
-            display="flex"
-            flexShrink={0}
-            minWidth={0}
-            borderRadius={1}
-            overflow="hidden"
-          />
-        </Grid2>
-        <Grid2 xs={10}>
-          <CardContent
-            sx={{
-              pt: 1,
-              paddingBottom: '0 !important',
-              minWidth: 0,
-            }}
-          >
-            <Typography
-              fontWeight="bold"
-              gutterBottom
-              variant="body1"
-              component="div"
-              width="100%"
-              noWrap
-            >
-              {name ?? <Skeleton />}
-            </Typography>
-            <Typography
-              variant="body2"
-              color="text.secondary"
-              width="100%"
+  <Tooltip title={description || ''}>
+    <Card
+      sx={{
+        width: '100%',
+        maxHeight: '80px',
+        flex: '1 1 calc(50% - 8px)',
+        outline: selected ? '2px solid #5050d2' : '',
```

Comment:
Use theme for the color.
Something like `theme.palette.primary.main` where `theme = useTheme`